### PR TITLE
[FUSEQE-12454] - tests for auditing

### PR DESCRIPTION
**Pull request by mkralik3**
//test: `@auditing`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Test auditing feature | Test auditing on API connector 
Test auditing feature | Test auditing on integration 
Test auditing feature | Test auditing on oauth 
```

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Integration - Database | Smoke - Periodic invocation to Insert 
Connection - CRUD | Create and delete 
Connection - CRUD | Kebab menu 
Connection - CRUD | Kebab menu edit and view 
Integration - DB to DB | Read and update operations 
Test auditing feature | Test auditing on API connector 
Test auditing feature | Test auditing on connection 
Test auditing feature | Test auditing on integration 
Test auditing feature | Test auditing on oauth 
Integration info | Check error for integration with the same name 
Integration info | Check error for integration which has same name after update 
```

**Comment by mkralik3** [comment]
retest this please

File: operator-tests/src/test/resources/operator/spec/auditing.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+apiVersion: syndesis.io/v1beta2
```
Comment (avano) [inline operator-tests/src/test/resources/operator/spec/auditing.yml]: this file should be in `spec/components/server/`

**Review by avano — DISMISSED** [review]

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Test auditing feature | Test auditing on API connector 
Test auditing feature | Test auditing on connection 
```

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Test auditing feature | Test auditing on integration 
```

**Review by avano — APPROVED** [review]

**Comment by mkralik3** [comment]
retest this please

**Comment by mkralik3** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**